# Generated from: /home/latty/link_dir/2015_li3ds/__ROS__/Ellipse_N/ROS/overlay_ws/src/sbg_ros_driver/sbgECom/src/binaryLogs/sbgEComBinaryLogEkf.h
# SbgLogEkfEulerData
uint32 timeStamp
float32[3] euler
float32[3] eulerStdDev
uint32 status